docs/PokemonCreation.md

```diff
@@ -0,0 +1,68 @@
+## Pokemon Creation
+
+Pokemon creation in FullScreenPokemon consists of three different types, Unit Test Pokemon creation, Trainer Battle Pokemon creation, and Wild Pokemon Grass Encounter creation.
+The two main files that handle Pokemon creation are [`Equations`](../src/components/Equations.ts) and [`Utilities`](../src/components/Utilities.ts),both of which incorporate ['numberMaker'](../node_modules/numbermakr/lib) for random generation of numbers.
+[`Equations`](../src/components/Equations.ts) includes functions for choosing random wild Pokemon and for creating the actual Pokemon object. [`Utilities`](../src/components/Utilities.ts) facilitates the creating of a wild Pokemon by preparing parameters than calling [`Equations`](../src/components/Equations.ts) creation of the Pokemon object.
+
+A newly created Pokemon has members of types:
+* experience: number
+* level: number
+* ev: IValuePoints
+* item?: string[]
+* iv: IValuePoints
+* raisedCriticalHitProbability?: boolean
+* statistics: IPokemonStatistics
+* status?: IStatus
+* types: string[]
+
+### Unit Test and Trainer Battles
+
+The steps for creating a Pokemon in a Unit Test and Trainer Battles are:
+    1. Pass into the newPokemon(chosenInfo: INewPokemon) function from [`Equations`](../src/components/Equations.ts) an on object with members:
+        * level: number
+        * title: string[]
+    2. Catch the return value in a variable of type IPokemon.
+
+```javascript
+//Create Pokemon in Unit Test and for Trainer Battle.
+const pokemonTitle: string[] = "CHARMANDER".split("");
+const pokemonLevel: number = 10;
+const pokemon: IPokemon = fsp.equations.newPokemon({
+    item: undefined, // || item as string[]
+    level: undefined || pokemonLevel,
+    moves: undefined, // || array type of IMove
+    title: pokemonTitle
+});
+```
+
+### Wild Pokemon Grass Encounter
+
+The steps for creating a Pokemon in a Wild Pokemon Grass Encounter are:
+1. The chooseWildPokemonForBattle(grass: Ithing) function is run.
+    * This function returns the Pokemon object by using the grass IMap and        IArea to determine what wild Pokemon can be encountered.
+2. After selecting valid wild Pokemon that can be encountered                      chooseRandomWildPokemon(options: IWildPokemonSchema[]) of return type           IWildPokemonSchema is called and chooses a Pokemon from the list of valid       Pokemon.
+3. Next, createPokemon(schema: IWildPokemonSchema) of return type IPokemon is      called which sets up an object of type INewPokemon which includes members:
```

Comment:
I'm seeing a lot of spaces here?

![image](https://user-images.githubusercontent.com/3335181/27108957-de2c8ee2-5054-11e7-87c6-ae1741fbbbe5.png)